cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/AppMetadataStore.java

```diff
@@ -650,12 +696,12 @@ private void addWorkflowNodeState(ProgramRunId programRunId, Map<String, String>
       return;
     }
 
-    // Use the actual runID from DB
-    workflowRunId = record.getProgramRunId();
-    runRecordFields = getProgramRunInvertedTimeKey(TYPE_RUN_RECORD_ACTIVE, workflowRunId,
-                                                   RunIds.getTime(workflowRun, TimeUnit.SECONDS));
+    // Use the actual ProgramRunId from DB
```

Comment:
why are these changes being done? It seems like just renaming variables?

Reply:
It's renaming variables plus I wanted not to override the existing one, which makes it very easy to ignore the value change.